src/App.js

```diff
@@ -1,11 +1,96 @@
-import './App.css';
+// modules
+import axios from 'axios';
+import { useState, useEffect } from 'react';
+// components
+import PageHeading from "./components/PageHeading.js";
+import DisplayPhotos from './components/DisplayPhotos.js';
+import Button from './components/Button.js';
+import PageFooter from './components/PageFooter.js';
+// // styling
+import './styles/sass/App.scss';
+
 
 function App() {
+
+  // useState objects
```

Comment:
Great use of state to generate your images and quotes! Something I did when splitting up my app was pass my stateful variables through props to my other components so I'd have access to the data I'd get back from the API call.